docs/docs/segment-terraform.md

```diff
@@ -0,0 +1,24 @@
+---
+id: terraform
+title: Terraform Context
+sidebar_label: Terraform
+---
+
+## What
+
+Display the currently active Terraform Workspace name.
+
+## Sample Configuration
+
+```json
+{
+  "type": "terraform",
+  "style": "powerline",
+  "powerline_symbol": "",
```

Comment:
Can you change this to `"powerline_symbol": "\uE0B0",`, I changed all docs to use UTF32 text representations rather than icons which can't always be rendered